.gitignore

```diff
@@ -1,8 +1,6 @@
 *.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
```

Comment:
Actually, I'm a little bit confused about this removal. I'm not saying you are not right. But can you explain why?